commit 2265675efb5a782e2bdec736a985f58c989c7226
Author: Harald Sitter <sitter@kde.org>
Date:   Mon Jan 5 02:20:35 2015 +0100

    rename kpackagetool man to kpackagetool5
    
    this aligns the manpage name to the binary name. also adjusted all
    references to the binary to use the 5 suffix.
    
    REVIEW: 121834

diff --git a/docs/kpackagetool/CMakeLists.txt b/docs/kpackagetool/CMakeLists.txt
index 346982d..1d97dd7 100644
--- a/docs/kpackagetool/CMakeLists.txt
+++ b/docs/kpackagetool/CMakeLists.txt
@@ -1,2 +1,2 @@
-kdoctools_create_manpage(man-kpackagetool.1.docbook 1 INSTALL_DESTINATION ${MAN_INSTALL_DIR})
+kdoctools_create_manpage(man-kpackagetool5.1.docbook 1 INSTALL_DESTINATION ${MAN_INSTALL_DIR})
 
diff --git a/docs/kpackagetool/man-kpackagetool.1.docbook b/docs/kpackagetool/man-kpackagetool.1.docbook
deleted file mode 100644
index 8f6fe5a..0000000
--- a/docs/kpackagetool/man-kpackagetool.1.docbook
+++ /dev/null
@@ -1,117 +0,0 @@
-<?xml version="1.0" ?>
-<!DOCTYPE refentry PUBLIC "-//KDE//DTD DocBook XML V4.5-Based Variant V1.1//EN" "dtd/kdedbx45.dtd" [
-<!ENTITY % English "INCLUDE">
-]>
-
-<refentry lang="&language;">
-<refentryinfo>
-<title>KPackage Manager</title>
-<date>2014-04-25</date>
-<releaseinfo>5.0</releaseinfo>
-<productname>&kde; kpackagetool</productname>
-</refentryinfo>
-
-<refmeta>
-<refentrytitle><command>kpackagetool</command></refentrytitle>
-<manvolnum>1</manvolnum>
-</refmeta>
-
-<refnamediv>
-<refname><command>kpackagetool</command></refname>
-<refpurpose>Install, list, remove Plasma packages</refpurpose>
-</refnamediv>
-
-<refsynopsisdiv>
-<cmdsynopsis>
-<command>kpackagetool</command>
-
-<group choice="opt"><option>--hash</option> <replaceable> path</replaceable></group>
-<group choice="opt"><option>-g, --global</option></group>
-<group choice="opt"><option>-t, --type</option> <replaceable> type</replaceable></group>
-<group choice="opt"><option>-i, --install</option> <replaceable> path</replaceable></group>
-<group choice="opt"><option>-s, --show</option> <replaceable> name</replaceable></group>
-<group choice="opt"><option>-u, --upgrade</option> <replaceable> path</replaceable></group>
-<group choice="opt"><option>-l, --list</option></group>
-<group choice="opt"><option>--list-types</option></group>
-<group choice="opt"><option>-r, --remove</option> <replaceable> name</replaceable></group>
-<group choice="opt"><option>-p, --packageroot</option> <replaceable> path</replaceable></group>
-<arg choice="opt">KDE Generic Options</arg>
-<arg choice="opt">Qt Generic Options</arg>
-</cmdsynopsis>
-</refsynopsisdiv>
-
-<refsect1>
-<title>Description</title>
-<para><command>kpackagetool</command> is a command line tool to
-install, list, remove Plasma packages.</para>
-
-</refsect1>
-
-<refsect1>
-<title>Options</title>
-
-<variablelist>
-<varlistentry>
-<term><option>--hash</option> <replaceable> path</replaceable></term>
-<listitem><para>Generate a SHA1 hash for the package at <quote>path</quote>.</para></listitem>
-</varlistentry>
-<varlistentry>
-<term><option>-g, --global</option></term>
-<listitem><para>For install or remove, operates on packages installed for all users.</para></listitem>
-</varlistentry>
-<varlistentry>
-<term><option>-t, --type</option> <replaceable> type</replaceable></term>
-<listitem><para>The type of package, &eg; theme, wallpaper, plasmoid, dataengine, runner, layout-template, &etc; [plasmoid].</para></listitem>
-</varlistentry>
-<varlistentry>
-<term><option>-i, --install</option> <replaceable> path</replaceable></term>
-<listitem><para>Install the package at <quote>path</quote>.</para></listitem>
-</varlistentry>
-<varlistentry>
-<term><option>-s, --show</option> <replaceable> name</replaceable></term>
-<listitem><para>Show information of package <quote>name</quote>.</para></listitem>
-</varlistentry>
-<varlistentry>
-<term><option>-u, --upgrade</option> <replaceable> path</replaceable></term>
-<listitem><para>Upgrade the package at <quote>path</quote>.</para></listitem>
-</varlistentry>
-<varlistentry>
-<term><option>-l, --list</option></term>
-<listitem><para>List installed packages.</para></listitem>
-</varlistentry>
-<varlistentry>
-<term><option>--list-types</option></term>
-<listitem><para>Lists all known Package types that can be installed.</para></listitem>
-</varlistentry>
-<varlistentry>
-<term><option>-r, --remove</option> <replaceable> name</replaceable></term>
-<listitem><para>Remove the package named <quote>name</quote>.</para></listitem>
-</varlistentry>
-<varlistentry>
-<term><option>-p, --packageroot</option> <replaceable> path</replaceable></term>
-<listitem><para>Absolute path to the package root. If not supplied, then the standard data 
-directories for this &kde; session will be searched instead.</para></listitem>
-</varlistentry>
-
-</variablelist>
-
-</refsect1>
-
-<refsect1>
-<title>See Also</title>
-
-<para>More detailed user documentation is available from <ulink
-url="help:/plasma-desktop">help:/plasma-desktop</ulink>
-(either enter this <acronym>URL</acronym> into &konqueror;, or run
-<userinput><command>khelpcenter</command>
-<parameter>help:/plasma-desktop</parameter></userinput>).</para>
-
-</refsect1>
-
-<refsect1>
-<title>Authors</title>
-<para><command>kpackagetool</command> was written by
-<personname><firstname>Aaron</firstname><surname>Seigo</surname></personname> <email>aseigo@kde.org</email>.</para>
-</refsect1>
-
-</refentry>
diff --git a/docs/kpackagetool/man-kpackagetool5.1.docbook b/docs/kpackagetool/man-kpackagetool5.1.docbook
new file mode 100644
index 0000000..47265ad
--- /dev/null
+++ b/docs/kpackagetool/man-kpackagetool5.1.docbook
@@ -0,0 +1,117 @@
+<?xml version="1.0" ?>
+<!DOCTYPE refentry PUBLIC "-//KDE//DTD DocBook XML V4.5-Based Variant V1.1//EN" "dtd/kdedbx45.dtd" [
+<!ENTITY % English "INCLUDE">
+]>
+
+<refentry lang="&language;">
+<refentryinfo>
+<title>KPackage Manager</title>
+<date>2014-04-25</date>
+<releaseinfo>5.0</releaseinfo>
+<productname>&kde; kpackagetool5</productname>
+</refentryinfo>
+
+<refmeta>
+<refentrytitle><command>kpackagetool5</command></refentrytitle>
+<manvolnum>1</manvolnum>
+</refmeta>
+
+<refnamediv>
+<refname><command>kpackagetool5</command></refname>
+<refpurpose>Install, list, remove Plasma packages</refpurpose>
+</refnamediv>
+
+<refsynopsisdiv>
+<cmdsynopsis>
+<command>kpackagetool5</command>
+
+<group choice="opt"><option>--hash</option> <replaceable> path</replaceable></group>
+<group choice="opt"><option>-g, --global</option></group>
+<group choice="opt"><option>-t, --type</option> <replaceable> type</replaceable></group>
+<group choice="opt"><option>-i, --install</option> <replaceable> path</replaceable></group>
+<group choice="opt"><option>-s, --show</option> <replaceable> name</replaceable></group>
+<group choice="opt"><option>-u, --upgrade</option> <replaceable> path</replaceable></group>
+<group choice="opt"><option>-l, --list</option></group>
+<group choice="opt"><option>--list-types</option></group>
+<group choice="opt"><option>-r, --remove</option> <replaceable> name</replaceable></group>
+<group choice="opt"><option>-p, --packageroot</option> <replaceable> path</replaceable></group>
+<arg choice="opt">KDE Generic Options</arg>
+<arg choice="opt">Qt Generic Options</arg>
+</cmdsynopsis>
+</refsynopsisdiv>
+
+<refsect1>
+<title>Description</title>
+<para><command>kpackagetool5</command> is a command line tool to
+install, list, remove Plasma packages.</para>
+
+</refsect1>
+
+<refsect1>
+<title>Options</title>
+
+<variablelist>
+<varlistentry>
+<term><option>--hash</option> <replaceable> path</replaceable></term>
+<listitem><para>Generate a SHA1 hash for the package at <quote>path</quote>.</para></listitem>
+</varlistentry>
+<varlistentry>
+<term><option>-g, --global</option></term>
+<listitem><para>For install or remove, operates on packages installed for all users.</para></listitem>
+</varlistentry>
+<varlistentry>
+<term><option>-t, --type</option> <replaceable> type</replaceable></term>
+<listitem><para>The type of package, &eg; theme, wallpaper, plasmoid, dataengine, runner, layout-template, &etc; [plasmoid].</para></listitem>
+</varlistentry>
+<varlistentry>
+<term><option>-i, --install</option> <replaceable> path</replaceable></term>
+<listitem><para>Install the package at <quote>path</quote>.</para></listitem>
+</varlistentry>
+<varlistentry>
+<term><option>-s, --show</option> <replaceable> name</replaceable></term>
+<listitem><para>Show information of package <quote>name</quote>.</para></listitem>
+</varlistentry>
+<varlistentry>
+<term><option>-u, --upgrade</option> <replaceable> path</replaceable></term>
+<listitem><para>Upgrade the package at <quote>path</quote>.</para></listitem>
+</varlistentry>
+<varlistentry>
+<term><option>-l, --list</option></term>
+<listitem><para>List installed packages.</para></listitem>
+</varlistentry>
+<varlistentry>
+<term><option>--list-types</option></term>
+<listitem><para>Lists all known Package types that can be installed.</para></listitem>
+</varlistentry>
+<varlistentry>
+<term><option>-r, --remove</option> <replaceable> name</replaceable></term>
+<listitem><para>Remove the package named <quote>name</quote>.</para></listitem>
+</varlistentry>
+<varlistentry>
+<term><option>-p, --packageroot</option> <replaceable> path</replaceable></term>
+<listitem><para>Absolute path to the package root. If not supplied, then the standard data 
+directories for this &kde; session will be searched instead.</para></listitem>
+</varlistentry>
+
+</variablelist>
+
+</refsect1>
+
+<refsect1>
+<title>See Also</title>
+
+<para>More detailed user documentation is available from <ulink
+url="help:/plasma-desktop">help:/plasma-desktop</ulink>
+(either enter this <acronym>URL</acronym> into &konqueror;, or run
+<userinput><command>khelpcenter</command>
+<parameter>help:/plasma-desktop</parameter></userinput>).</para>
+
+</refsect1>
+
+<refsect1>
+<title>Authors</title>
+<para><command>kpackagetool5</command> was written by
+<personname><firstname>Aaron</firstname><surname>Seigo</surname></personname> <email>aseigo@kde.org</email>.</para>
+</refsect1>
+
+</refentry>
